charts/iperf3/templates/deployment.yml

```diff
@@ -3,7 +3,7 @@ kind: Deployment
 metadata:
   name: {{.Chart.Name}}
 spec:
-  replicas: 1
+  replicas: {{ .Values.replicas | default 1 }}
```

Comment:
please set the default in the values, not in the deployment via gotpl

Reply:
done